catalyst/contrib/datasets/__init__.py

```diff
@@ -1,4 +1,20 @@
 # flake8: noqa
 
+import logging
+
+logger = logging.getLogger(__name__)
```

Comment:
cause it's weird :)

Reply:
I have changed the order a little bit, is it fine now?